src/Catalyst.Core.Modules.Consensus.Tests/UnitTests/Deltas/DeltaBuilderTests.cs

```diff
@@ -113,220 +117,227 @@ public void BuildDeltaEmptyPoolContent()
+
+        //    var deltaBuilder = new DeltaBuilder(transactionRetriever, _randomFactory, _hashProvider, _peerSettings,
+        //        _cache, _dateTimeProvider, _mapperProvider, _logger);
+        //    var candidate = deltaBuilder.BuildCandidateDelta(_previousDeltaHash);
+
+        //    ValidateDeltaCandidate(candidate, _zeroCoinbaseEntry.ToByteArray());
+
+        //    _cache.Received(1).AddLocalDelta(Arg.Is(candidate), Arg.Any<Delta>());
+        //}
+
+        //todo
+        //[Fact]
+        //public void BuildDeltaCheckForAccuracy()
+        //{
+        //    var transactions = Enumerable.Range(0, 20).Select(i =>
+        //    {
+        //        var transaction = TransactionHelper.GetPublicTransaction(
+        //            (uint)i,
+        //            receiverPublicKey: i.ToString(),
+        //            transactionFees: (ulong)_random.Next(),
+        //            timestamp: _random.Next(),
+        //            signature: i.ToString());
+        //        return MempoolHelper.GetMempoolItems(transaction, _hashProvider);
+        //    }).SelectMany(x => x).ToList();
+
+        //    var transactionRetriever = BuildRetriever(transactions);
+        //    var selectedTransactions = BuildSelectedTransactions(transactions);
+        //    var publicEntries = selectedTransactions.Select(x => x.ToProtoBuff<MempoolItem, PublicEntry>(_mapperProvider));
+
+        //    var salt = BitConverter.GetBytes(_randomFactory.GetDeterministicRandomFromSeed(_previousDeltaHash.ToArray()).NextInt());
+
+        //    var rawAndSaltedEntriesBySignature = publicEntries.Select(e =>
+        //        {
+        //            var publicEntriesProtoBuff = e;
+        //            return new
+        //            {
+        //                RawEntry = publicEntriesProtoBuff,
+        //                SaltedAndHashedEntry =
+        //                    _hashProvider.ComputeMultiHash(publicEntriesProtoBuff.ToByteArray().Concat(salt))
+        //            };
+        //        });
+
+        //    var shuffledEntriesBytes = rawAndSaltedEntriesBySignature
+        //       .OrderBy(v => v.SaltedAndHashedEntry.ToArray(), ByteUtil.ByteListComparer.Default)
+        //       .SelectMany(v => v.RawEntry.ToByteArray())
+        //       .ToArray();
+
+        //    var expectedBytesToHash = BuildExpectedBytesToHash(selectedTransactions, shuffledEntriesBytes);
+
+        //    var deltaBuilder = new DeltaBuilder(transactionRetriever, _randomFactory, _hashProvider, _peerSettings,
+        //        _cache, _dateTimeProvider, _mapperProvider, _logger);
+        //    var candidate = deltaBuilder.BuildCandidateDelta(_previousDeltaHash);
+
+        //    ValidateDeltaCandidate(candidate, expectedBytesToHash);
+
+        //    _cache.Received(1).AddLocalDelta(Arg.Is(candidate), Arg.Any<Delta>());
+        //}
+
+        private static IDeltaTransactionRetriever BuildRetriever(List<PublicEntry> transactions)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unused private method 'BuildRetriever'.](https://app.codacy.com/app/catalyst-network/Catalyst.Framework/pullRequest?prid=4562326)